src/Parsing/OutputException.php

```diff
@@ -4,6 +4,8 @@
 
 /**
  * Thrown if the CSS parser attempts to print something invalid.
+ *
+ * @internal
  */
 class OutputException extends SourceException
```

Comment:
This is thrown on attempting to render a declaration block whose selector(s) have all been removed.  There's a test specifically for that situation and behaviour (with or without the 'throw exceptions' setting), so this doesn't look internal to me.

The same applies to the entire exception hierarchy.  Though I don't see a reason not to make them internal moving forwards..

Reply:
Done.